templates/todo/projects/nodejs-mongo/.repo/terraform/azure.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: todo-nodejs-mongo-terraform
+metadata:
+  template: todo-nodejs-mongo-terraform@0.0.1-beta

```

Comment:
Add an entry to Changelog about a new template was added?

Reply:
added

Reply:
not sure what the standard messages are. will resolve to merge now and update it later